src/content/protocols/ccip.md

```diff
@@ -0,0 +1,384 @@
+type: "Bridge"
+logo: "/images/infrastructure-logos/chainlink-logo.png"
+stage: "I0"
+reasons: []
+risks: ["L", "H", "L", "M", "L"]
+author: ["sagaciousyves"]
+submission_date: "2025-05-07"
+publish_date: "1970-01-01"
+update_date: "1970-01-01"
+---
+
+# Summary
+
+CCIP is a cross-chain protocol that allows any smart contract protocol to use a multi-purpose bridge to transfer messages and tokens from one blockchain to another.
+
+This report only covers scanned permissions and the owners of these permission of the cross-chain communication infrastructure between Ethereum Mainnet and Arbitrum based on smart contract analysis. The code of the oracle networks and centralization vectors thereof are not part of the scope.
+
+# Protocol Analysis
+
+There are three oracle networks that help to make CCIP as a bridge secure. Each network has a distinct role inside the cross chain communication flow.
+
+There is the _Risk Management Network_ which monitors the chain, and can halt the cross-chain flow of messages and tokens to and from a network. Additionally, the _Risk Management Network_ “blesses” or “curses” messages that should be moved cross chain. When the count of nodes blessing a message reaches the quorum, the message can be executed on the destination chain.
+
+Additionally, there is the _Committing Network_ which observes events on the source chain that signal messages that should be moved cross chain. If it detects such a transaction, the _Committing Network_ posts a root to the `CommitStore` contract on the destination chain.
+
+Lastly, there is the _Executing Network_ that observes the transactions and if roots are posted to the `CommitStore` and the roots are blessed, the executing network triggers the message to be executed on the target chain with the blessed messages.
+
+![Onchain Infrastructure](./diagrams/CCIP_Infrastructure.png)
+
+# Centralization
+
+## Governance
+
+The smart contracts of the CCIP protocol are governed by the `RBACTimelock` contract. The governance can change parameters and the onchain addresses of the nodes in each network (Risk, Committing, Executing).
+
+Governance actions are proposed by multichain multisigs (MCMSs) which have the Proposer role by calling `scheduleBatch` on the `RBACTimelock` contract (see [role owner table](#role-owners-rbactimelock)). The `RBACTimelock` contract enforces a minimum timelock, which is set to 3 hours. Particpants of governance can choose a specific timelock which is above or equal to the minimum timelock. The scheduled transactions are then ready to be executed after the timelock has passed. During the time of the exit window the scheduled transactions is object to cancellation by MCMSs with the Canceller role. With no cancellation during the exit window period execution becomes permissionless.
+
+A Bypasser role is assigned to a Multichain Multisig (MCMS) for the case of an emergency to execute transactions instantaneously without the possibility to cancel the transaction.
```

Comment:
Here link to the security council table?


```suggestion
A Bypasser role is assigned to a Multichain Multisig (MCMS) in case of an emergency to execute transactions instantaneously.
```